Network Inspector example has an outdated Docker image.

**URL of the page in question:**
https://docs.filecoin.io/build/examples/network-inspector/step-1-start-lotus-devnet-and-go-ipfs/#steps

**What's wrong with this page?**
When running `make devnet BIGSECTORS=true` it spins up 2 containers

The container `devnet_lotus_1` crashes:


```
2020-09-03T15:57:42.378Z	INFO	devnet-main	app/main.go:42	Starting devnet with speed = 1500, numminers = 1, bigsectors = true, ipfsaddr = /dns4/ipfs/tcp/5001
2020-09-03T15:57:42.384Z	WARN	chainstore	store/store.go:136	no previous chain state found
2020-09-03T15:57:42.384Z	WARN	genesis	testing/genesis.go:36	Generating new random genesis block, note that this SHOULD NOT happen unless you are setting up new network
init set t3qskgjhkzf4jk2nw52bkryccbxg5rt3blg53ayv5cgmfewwkrj6m3ogk6ooixspm7xpru56hds3nzvbuc6myq t0100
publishing 2 storage deals on miner t01000 with worker t3qskgjhkzf4jk2nw52bkryccbxg5rt3blg53ayv5cgmfewwkrj6m3ogk6ooixspm7xpru56hds3nzvbuc6myq
Added weight to claim:  536870912 5368709121
Added weight to claim:  1073741824 10737418241
2020-09-03T15:57:42.399Z	INFO	genesis	genesis/genesis.go:331	Empty Genesis root: bafy2bzaceaa43et73tgxsoh2xizd4mxhbrcfig4kqp25zfa5scdgkzppllyuu
2020-09-03T15:57:42.401Z	WARN	chainstore	store/store.go:347	no heaviest tipset found, using [bafy2bzacecmleg6tl6r74iwe3oqok4y5hfjzycvcriikbqxkwakkvnsif3rcs]
2020-09-03T15:57:42.401Z	INFO	chainstore	store/store.go:352	New heaviest tipset! [bafy2bzacecmleg6tl6r74iwe3oqok4y5hfjzycvcriikbqxkwakkvnsif3rcs] (height=0)
2020-09-03T15:57:42.404Z	INFO	p2pnode	lp2p/addrs.go:114	Swarm listening at: [/ip4/0.0.0.0/tcp/0 /ip6/::/tcp/0]
2020-09-03T15:57:42.404Z	INFO	dht/RtRefreshManager	rtrefresh/rt_refresh_manager.go:265	starting refreshing cpl 0 with key  �/ (routing table size was 0)
2020-09-03T15:57:42.404Z	WARN	dht/RtRefreshManager	rtrefresh/rt_refresh_manager.go:128	failed when refreshing routing table2 errors occurred:
	* failed to query for self, err=failed to run refresh DHT query for key= V�J�p��I87�
fwؙ¯T!��C�G&#5�, err=failed to find any peer in table
	* failed to refresh cpl=0, err=failed to run refresh DHT query for key= �/, err=failed to find any peer in table


panic: starting node: could not build arguments for function "github.com/filecoin-project/lotus/node/modules".HandleIncomingBlocks (/go/pkg/mod/github.com/filecoin-project/lotus@v0.4.0/node/modules/services.go:61): failed to build *pubsub.PubSub: could not build arguments for function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12): failed to build dtypes.DrandBootstrap: function "reflect".makeFuncStub (/usr/local/go/src/reflect/asm_amd64.s:12) returned a non-nil error: lookup _dnsaddr.pl-sin.testnet.drand.sh on 127.0.0.11:53: no such host

goroutine 1 [running]:
main.main()
	/app/main.go:46 +0x5e0
```

**What would you like to have changed/amended/fixed in order to make things right?**
Need to update `textile/lotus-devnet:v0.4.0` image to `textile/lotus-devnet:v0.5.3`

**Would you be interested in helping fix this page?**
\-

--- a/docs/build/examples/network-inspector/step-1-start-lotus-devnet-and-go-ipfs.md
+++ b/docs/build/examples/network-inspector/step-1-start-lotus-devnet-and-go-ipfs.md
@@ -34,7 +34,7 @@ version: '3.7'
 
 services:
   lotus:
-    image: textile/lotus-devnet:v0.4.0
+    image: textile/lotus-devnet:v0.5.3
     ports:
       - 7777:7777
     environment:

--- a/docs/build/examples/network-inspector/step-2-run-the-react-app.md
+++ b/docs/build/examples/network-inspector/step-2-run-the-react-app.md
@@ -23,7 +23,7 @@ git checkout local
 2. Start the app.
 
 ```bash
-npm install
+npm ci
 npm start
 ```
 

